src/asyncresults.jl

```diff
@@ -102,9 +98,11 @@ function _consume(jl_conn::Connection)
                 _cancel(jl_conn)
             end
 
-            last_log == curr && debug(LOGGER, "Waiting to read from connection $(jl_conn.conn)")
+            last_log == curr &&
+                debug(LOGGER, "Waiting to read from connection $(jl_conn.conn)")
             wait(watcher)
-            last_log == curr && debug(LOGGER, "Consuming input from connection $(jl_conn.conn)")
+            last_log == curr &&
+                debug(LOGGER, "Consuming input from connection $(jl_conn.conn)")
```

Comment:
This change is also maybe not right?
given it's over the line limit, I think we'd maybe prefer it to have the line broken up like
```julia
last_log == curr && debug(
    LOGGER, "Consuming input from connection $(jl_conn.conn)"
)
```
or something like that?

Reply:
I think my suggestion here is in-keeping with BlueStyle preference for assignment e.g.
```julia
# Yes
some_big_long_variable_that_takes_up_most_of_the_line_but_still_leaves_room_for_f = f(
    "argument doesn't fit on that first line", 2
)

# No
some_big_long_variable_that_takes_up_most_of_the_line_but_still_leaves_room_for_f =
    f("argument doesn't fit on that first line", 2)
```

Reply:
fyi @domluna

Reply:
Actually, I think we may prefer

```julia
if last_log == curr
    debug(LOGGER, "Consuming input from connection $(jl_conn.conn)")
end
```

I think `&&`-as-control-flow should only be used if it can be done in one line.

Reply:
makes sense to me

and matches our preference for ternary operators to only be used if they fit on one line

Reply:
I have no say on blue style but Eric's suggestion makes more sense since it's in line with the conditionals transform you want to do

Reply:
yeah, sounds good to me!

Reply:
https://github.com/invenia/BlueStyle/issues/72